tests/app/dao/test_provider_details_dao.py

```diff
@@ -136,14 +204,14 @@ def test_get_alternative_sms_provider_fails_if_unrecognised():
     ({'mmg': 50, 'firetext': -100}, {'mmg': 40, 'firetext': -90}),
 ])
 def test_reduce_sms_provider_priority_switches_provider(
```

Comment:
Optional: It it worth changing this test name now? Seems to more reflect older behaviour